.gitignore

```diff
@@ -22,4 +22,3 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 
-.env
```

Comment:
I think the .env file shouldn't be commited.

Reply:
Actually I noticed that when I cloned the repo, there were changes in the .env file that contained the API keys variable (not the keys themselves) for open weather api and google maps api. For spotify, there are two keys to be used in the code so thats why I removed the value of the keys but commited that file so that when somebody else pulls the changes, they dont have to create those variables separately. Let me know what can be done finally.

Reply:
Makes sense. Maybe @NoahCardoza could help clarify that?

Reply:
@NoahCardoza actually the API keys weren't working when the file name was example.env. So, my partner and I were working around the error, so we were just trying out different things and changing the name from example.env to just .env worked. Thats why, had to do that. I know this doesn't make any sense but yeah, kindly let me know what can be done finally.